katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-#global prever .rc1
+# %%global prever .rc1
```

Comment:
Is this like a double negative? I thought double `%%` was used to reference a macro or create a macro?

I assume that `#% %global` won't work?